mm2src/ordermatch_tests.rs

```diff
@@ -1802,6 +1808,7 @@ fn test_request_and_fill_orderbook() {
         assert_eq!(actual, expected_request);
 
         let result = orders
+            .clone()
```

Comment:
I think you can avoid cloning here and fill the `conf_infos` in the `map` closure. Maybe there's another way to avoid `clone`.